rest_framework/authtoken/serializers.py

```diff
@@ -18,13 +19,25 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _('User account is disabled.')
-                    raise serializers.ValidationError(msg)
+                    raise serializers.ValidationError(
+                        ValidationErrorMessage(
+                            msg,
+                            code='authorization')
```

Comment:
I did a change on the api to make it more like the django api.
Good point about the forms API, i'll look into that.

Just to clarify, are you against the fact that we put the `code` on the string-like object?
Because right now I don't quite see how we could maintain backward compatibility without that.